salt/utils/path.py

```diff
@@ -41,108 +30,14 @@ def islink(path):
     """
     Equivalent to os.path.islink()
     """
-    if six.PY3 or not salt.utils.platform.is_windows():
-        return os.path.islink(path)
```

Comment:
**question**: does os.path.islink() work on Windows now? Looks like this was doing some windows specific shenanigans...

Reply:
Yes, `islink` works on windows now.